cmd/grpc-client-cli/app.go

```diff
@@ -142,8 +153,10 @@ func (a *app) callService(method *desc.MethodDescriptor, message []byte) error {
 			err = a.callServerStream(ctx, method, selectedMsg)
 		} else if !method.IsServerStreaming() && !method.IsClientStreaming() {
 			err = a.callUnary(ctx, method, selectedMsg)
+		} else if !method.IsServerStreaming() && method.IsClientStreaming() {
+			err = a.callClientStream(ctx, method, messages)
 		} else {
-			err = errors.New("client/bi-directional streaming is not supported")
+			err = errors.New("bi-directional streaming is not supported")
 		}
```

Comment:
Technically we check `!method.IsServerStreaming()` twice? But also use two different styles (lines 137 vs this)

Maybe nest the seccond check `!method.isClientStreaming()` and `method.isClientStreaming()`?

Reply:
ok, check now, restructure a bit